pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mango-agents"
+version = "2.1.0"
+authors = [
+  { name="mango Team", email="mango@offis.de" },
+]
+description = "Modular Python Agent Framework"
+readme = "readme.md"
+requires-python = ">=3.10"
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
```

Comment:
though the mango codebase works with 3.13, note that currently this fails due to msgspec not being installable.

Especially the GIL-removal (aka freethreading) part is missing yet.
See https://github.com/jcrist/msgspec/issues/744

Reply:
Yes true, I tested it before that ;) works fine!